dataline-config/src/main/resources/json/StandardSyncInput.json

```diff
@@ -23,6 +23,10 @@
     },
     "state": {
       "$ref": "State.json"
+    },
+    "standardSyncSummary": {
```

Comment:
Why do you need this?
Can you rename it so it matches the description? `previousSyncSummary` or smth like that

Reply:
conceptually i've been thinking about standardSyncSummary as our version of the state file. the idea being that there is some standard set of data that every sync gets to know about the previous run. i think at least theoretically this idea is appealing to me. functionally it's not doing much right now. we can make the following calls:
1. we like the concept of `standardSyncSummary` as our standard version of the state file.
1. we don't like it and never plan to pass it in
1. we don't pass it in for now because it's not really used and we can always use it later.

Reply:
i'm amenable to any of these options.

Reply:
If we aren't using it right now then we shouldn't add it.